tests/sentry/replays/test_project_replay_clicks_index.py

```diff
@@ -0,0 +1,225 @@
+import datetime
+from unittest import mock
+from uuid import uuid4
+
+import pytest
+from django.urls import reverse
+
+from sentry.replays import tasks
+from sentry.replays.testutils import mock_replay, mock_replay_click
+from sentry.testutils import APITestCase, ReplaysSnubaTestCase
+from sentry.testutils.silo import region_silo_test
+from sentry.utils import kafka_config
+
+REPLAYS_FEATURES = {"organizations:session-replay": True}
+
+
+@pytest.fixture(autouse=True)
+def setup():
+    with mock.patch.object(kafka_config, "get_kafka_producer_cluster_options"):
+        with mock.patch.object(tasks, "KafkaPublisher"):
+            yield
+
+
+@region_silo_test
+class OrganizationReplayDetailsTest(APITestCase, ReplaysSnubaTestCase):
```

Comment:
looking at the codecov https://app.codecov.io/gh/getsentry/sentry/pull/47332 ,  could we add a test for:
- a paren expression? https://app.codecov.io/gh/getsentry/sentry/pull/47332#503bea6c3978007e2abd012d68296f82-R249
- i'm also confused but it seems like its missing a case where theres only one condition? https://app.codecov.io/gh/getsentry/sentry/pull/47332#503bea6c3978007e2abd012d68296f82-R206